lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -213,6 +213,8 @@ def purchase(money, payment_method, options = {})
       end
 
       def capture(money, intent_id, options = {})
+        return Response.new(false, 'Only Authorizations performed via the Payment Intent API are  capturable') unless intent_id.include?('pi_')
```

Comment:
If you could just remove that extra space between are and capturable

Reply:
Everything else looks good

Reply:
done!